cfme/tests/containers/test_containers_properties.py

```diff
@@ -66,6 +71,10 @@ def test_routes_properties_rel(provider, rel):
                           'resource_version'
                           ])
 def test_projects_properties_rel(provider, rel):
+    """ This module verifies data integrity in the Properties table for
+        projects, all cells in the Properties table should contain
+        valid data
+    """
```

Comment:
@mfalesni thanks, will change it right now. Still getting used to python commenting style:)